bootstrap

```diff
@@ -177,7 +177,20 @@ if [ "${cpu_partitioning}" == "1" ]; then
     if [ "${endpoint}" == "k8s" ]; then
 	environment="container"
     fi
-    /usr/local/bin/discover-cpu-partitioning.py --environment=${environment}
+    cpu_data=$(/usr/local/bin/discover-cpu-partitioning.py --environment=${environment})
+
+    # log the data returned
+    echo -e "${cpu_data}"
+
+    housekeeping_cpus=$(echo -e "${cpu_data}" | grep housekeeping | awk '{ print $3 }')
+    export HK_CPUS=${housekeeping_cpus}
+
+    isolated_cpus=$(echo -e "${cpu_data}" | grep isolated | awk '{ print $3 }')
+    export WORKLOAD_CPUS=${isolated_cpus}
+
+    # pin myself to the housekeeping cpus so that I and my children
+    # are contained there
+    taskset --cpu-list --pid ${HK_CPUS} $$
 else
     echo "Disabled"
```

Comment:
"cpu partitioning disabled" ? also on debug mode only ?

Reply:
Same as above.

This ends up looking something like this: